### ISSUE TITLE
v2.0.16 source package doesn't contain flawfinder py

### ISSUE THREAD

**Issue by OlHermit** [david-a-wheeler/flawfinder#47]
Downloaded source package from https://dwheeler.com/flawfinder/flawfinder-2.0.16.tar.gz as directed by website https://dwheeler.com/flawfinder/ 

Was trying to understand why on windows the `<pythondir>/scripts/flawfinder` file was now `<pythondir>/scripts/flawfinder.exe` 
 (Should this be an issue as well, or is this expected?

Found that the 2.0.16 source download doesn't contain `flawfinder` or `flawfinder` script at all.

Expect this was due to recent refactor, which left this behind in the makefile:

```makefile
distribute: clean flawfinder.pdf flawfinder.ps
	...<snip>...
	# We don't need both "flawfinder" and "flawfinder.py":
	rm -f ,tempdir/flawfinder.py
```
which removes `flawfinder.py` (which in v2.0.15 was named just `flawfinder`

**Comment by david-a-wheeler** [comment]
Argh! You're absolutely right. Thanks for the report!

Time for a new release :-).

**Comment by david-a-wheeler** [comment]
THANK YOU AGAIN for your report.

I've released a new version 2.0.17, after removing the offending makefile line you pointed out. I made a few tweaks while I was at it, but the main issue was to fix that.

I *think* it's okay now, so I'm closing this issue.

If there's *still* a problem, or some other problem, PLEASE let me know and we'll re-open this.

**Comment by david-a-wheeler** [comment]
I *think* the weird ".exe" was a side-effect of not having the correct file there at all.

If you find other problems, please just open a new issue, so that I can track it effectively.

THANK YOU!!
